lib/cosmos/config/config_parser.rb

```diff
@@ -146,6 +146,7 @@ def error(message, usage = "", url = @url)
 
     # Called by the ERB template to render a partial
     def render(template_name, options = {})
+      raise Error.new(self, "Partial name '#{template_name}' must begin with an underscore.") if template_name[0] != '_'
```

Comment:
Should this use File.basename to support absolute paths?

Reply:
Nevermind, it looks like below it enforces the partial be in the same folder.  :+1: